src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/Microsoft.DotNet.Maestro.Tasks.Tests.csproj

```diff
@@ -16,5 +16,47 @@
     <ProjectReference Include="..\Microsoft.DotNet.Maestro.Tasks\Microsoft.DotNet.Maestro.Tasks.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="TestManifests\BadXml\BadXml.xml">
```

Comment:
I'd probably just do `TestManifests\**`.  But for most of these, I would much, _much_ rather they be strings inside the tests themselves, so that I don't have root around the file system to figure out what a test is doing or why it's failing.

Reply:
I like the refactoring idea, so I'm pursuing that as my first choice to get this done. Without refactoring I'd have to mock Directory, FileStream & XmlSerializer, so it seems simpler to factor out the logic that uses the parsed xml.

Reply:
Yeah, I'd rip the main method in half.  Do all the "parsy" stuff first, then past the parsed XML/objects to some secondary method. Then just test that method.

Reply:
Done - split the file loading from the object parsing.